introduction_to_amazon_algorithms/jumpstart_semantic_segmentation/Amazon_JumpStart_Semantic_Segmentation_Extract_Image.ipynb

```diff
@@ -0,0 +1,575 @@
+{
```

Comment:
<p>Use present tense</p><p> </p><p>we will overlay-&gt; we overlay</p><p>you will find -&gt; you can find</p><p> </p><p>Also, add a comma and some grammar things (i.e. which, the)</p><p> </p><p>After executing below cell, you will find the Final_pedestrian.jpg in your folder that is extracted image of the pedestrians.  -&gt; After executing below cell, you will find the Final_pedestrian.jpg in your folder, which is the extracted image of the pedestrians.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>